Gruntfile.coffee

```diff
@@ -494,6 +508,28 @@ module.exports = (grunt) ->
               options:
                 dir: 'build/coverage/generic_ui'
 
+    jasmine_chromeapp: {
+      all: {
+        src: ['node_modules/freedom-for-chrome/freedom-for-chrome.js',
+              'build/compile-src/integration/scripts/uproxy.js',
+              'build/compile-src/integration/scripts/consent.js',
+              'build/compile-src/integration/gtalk_credentials.js',
+              'build/compile-src/integration/**/*.js',
+              'build/compile-src/integration/**/*.json',
+              'build/compile-src/integration/core.spec.js']
+        options: {
+          paths: ['node_modules/freedom-for-chrome/freedom-for-chrome.js',
+                  'build/compile-src/integration/scripts/uproxy.js',
+                  'build/compile-src/integration/scripts/consent.js',
+                  'build/compile-src/integration/scripts/uproxy-lib/arraybuffers/arraybuffers.js',
+                  'build/compile-src/integration/gtalk_credentials.js',
+                  'build/compile-src/integration/core.spec.js'
+          ],
+          keepRunner: true,
+          binary: "/Applications/Google\ Chrome\ Canary.app/Contents/MacOS/Google\ Chrome\ Canary"
```

Comment:
This looks rather hard wired to Chrome Canary on Mac... will it work on linux? Or with other versions of Chrome?